applications/service_brokers/uninstalling-service-catalog.adoc

```diff
@@ -0,0 +1,22 @@
+[id="sb-uninstalling-service-catalog"]
+= Uninstalling Service Catalog
+include::modules/common-attributes.adoc[]
+:context: sb-uninstalling-service-catalog
+
+toc::[]
+
+[IMPORTANT]
+====
+Starting in {product-title} 4.4, the `service-catalog-controller-manager` and
+`service-catalog-apiserver` cluster Operators are now set to
+`Upgradeable=false`, meaning they will block future cluster upgrades to the next
+minor version, for example 4.5, if they are still installed at that time.
+Upgrading to z-stream releases such as 4.4.z, however, are still permitted in
+this state.
+====
+
+You can uninstall Service Catalog if you have installed it previously, which is
+required to upgrade to future minor versions of {product-title}.
```

Comment:
"which is required" 
It's difficult to know if this means uninstall or install is required. Assuming the former? I'd split into two sentences here, something like:

> You can uninstall Service Catalog if you have installed it previously. Uninstall is
> required to upgrade to future minor versions of {product-title}.

Reply:
Ooh yes, that would be a bad misunderstanding.